include/color_spinor_field_order.h

```diff
@@ -1184,31 +1201,39 @@ namespace quda
       using AllocInt = typename AllocType<huge_alloc>::type;
       using norm_type = float;
       Float *field = nullptr;
+#if 0
```

Comment:
Does it make sense to add a new cmake option for these?

Reply:
As discuss on slack, I've put the legacy code path behind a macro that can be enabled if ever need be.  [2bcff04](https://github.com/lattice/quda/pull/1465/commits/2bcff0487888b8b296b598a55c95048c9b441855)